pubspec.yaml

```diff
@@ -32,6 +32,12 @@ dependencies:
   provider: ^4.3.2
   adaptive_breakpoints: ^0.0.2
 
+  # TODO: After web_benchmarks PR merges, update to flutter/web_benchmarks.
```

Comment:
I think this should be in dev_dependencies

Reply:
Thank you!

Done.